internal/handler/v2/audit/audit_list_dataagreement_records.go

```diff
@@ -144,6 +146,13 @@ func AuditListDataAgreementRecords(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Sort the consentRecords based on timestamp
+	sort.SliceStable(consentRecords, func(i, j int) bool {
+		timeI, _ := time.Parse(time.RFC3339, consentRecords[i].Timestamp)
```

Comment:
Can you explain the solution ?

Reply:
Sort the consent records based on revision timestamp, by converting timestamp to time format.